index.js

```diff
@@ -44,10 +54,10 @@ function configureBot(bot) {
         }
     }
 
-    // The bot will announce whenever it collects a log or an apple
+    // The bot will announce whenever it collects a ore or an apple
     bot.on('playerCollect', async (collector, collected) => {
         const itemName = bot.getEntityName(collected).toLowerCase();
-        if (collector.username === bot.mineflayer().username && (itemName.includes('log') || itemName === 'apple')) {
+        if (collector.username === bot.mineflayer().username && (itemName.includes('ore') || itemName === 'apple')) {
```

Comment:
We don't need this anymore since collecting items automatically sends a message in chat

Reply:
you're correct that we don't need it, but it is a good example of how to respond to playerCollect events